kratos/sources/model_part.cpp

```diff
@@ -1786,6 +1787,10 @@ void ModelPart::PrintData(std::ostream& rOStream) const
     rOStream << "    Number of tables : " << NumberOfTables() << std::endl;
     rOStream << "    Number of sub model parts : " << NumberOfSubModelParts() << std::endl;
     if (!IsSubModelPart()) {
+        if (IsDistributed()) {
+            rOStream << "    Distributed over " << mpCommunicator->TotalProcesses() << " processes" << std::endl;
+            rOStream << "    Total number of processes: " << ParallelEnvironment::GetDataCommunicator("World").Size() << std::endl;
```

Comment:
@adityaghantasala I added it, this is the same as in the kernel:
https://github.com/KratosMultiphysics/Kratos/blob/457a15824bd40485fe612c69a56559a058dbdc10/kratos/sources/kernel.cpp#L143-L144

however this means that we have to include another header: `#include "includes/parallel_environment.h"`. I guess this is ok since it is a cpp but still...

**Alternative:**
this would be the alternative without additional includes:
```suggestion
            rOStream << "    Total number of processes: " << DataCommunicator::GetDefault().Size() << std::endl;
```

however someone could change the default `DataCommunicator` then this would be wrong

**Conclusion:**
I am not sure if we should add it. After all the number of processes is sth critical for an MPI-run, one does not easily "forget" that...
What do you think?